src/coreclr/tools/Common/Compiler/DependencyAnalysis/AssemblyStubNode.cs

```diff
@@ -68,6 +66,9 @@ public override ObjectData GetData(NodeFactory factory, bool relocsOnly)
                     arm64Emitter.Builder.AddSymbol(this);
                     return arm64Emitter.Builder.ToObjectData();
 
+                case TargetArchitecture.Wasm32:
```

Comment:
Would this lead to infinite recursion if this is actually reached? Maybe we can undo all changes tot his file.